### PR TITLE
Add RFC for jammy jellyfish OS

### PR DESCRIPTION
**Pull request by dsboulder**
Signed-off-by: David Stevenson <stevensonda@vmware.com>

### CODE REVIEW

File: toc/rfc/rfc-draft-jammy-os.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+The creation of a new set of cflinuxfs4-compatible buildpacks, which are hopefully very similar to the existing buildpacks, belongs to the *App Runtime Interfaces Working Group*. This could be an opportunity to switch to some implementation of Cloud Native Buildpacks, but we run the risk of exceeding the bionic timeline by extending the scope.
+
+@meyer owns this operation and can provide details needed on the buildpacks/rootfs side.
+Target start time: April 2022
+
+### Ensure all CF components are compatible with new stemcells, fix them as needed
+Many CF components should run unmodified on the new Operating System, but there are always some that need work. Some job control scripts typically end up executing OS-specific operations, any kernel performance tweaks need to be reassessed (gorouter!), and low-level jobs like garden typically need modification to be compatible.
+
+This work should be driven out by the CF-deployment conversion work, with *relevant working groups* fixing incompatibility issues when discovered.
+Target start time: June 2022 (using early stemcell builds)
```
Comment (rkoster) [inline toc/rfc/rfc-draft-jammy-os.md:46]: Jammy stemcell work is well underway. We expect to have an alpha build on bosh.io in the coming days. We also started to collect some docs to help release authors by documenting known incompatibilities: https://bosh.io/docs/jammy-migration/

One major thing to point out is the openssl bump to 3.0. Which seems to so for only be compatible with Ruby 3.1.

**Comment by dsboulder** [comment]
Wanted you folks to see this before I submit it to final 1-week review + decision period. Feel free to comment!
@rkoster @pburkholder @norman-abramovitz @bgandon @meyer

**Comment by dsboulder** [comment]
We haven't seen any discussion yet and work is really kicking off, so I'm starting the final review process and am hoping to close this out next Tuesday TOC meeting... unless discussion prohibits that.

**Comment by pburkholder** [comment]
Thanks for soliciting my input. On the cloud.gov side we have concerns about CF moving faster than Canonical can on getting FIPS validated versions of the OS's etc. However, a) going to Focal doesn't really help with the larger issues with FIPS kernel support b) getting to OpenSSL 3 (in Jammy) is a win. 

So: go for it!

**Review by Gerg — APPROVED** [review]

**Review by LeePorte — APPROVED** [review]

**Review by stephanme — APPROVED** [review]

**Comment by dsboulder** [comment]
Ready to approve this! I'll follow the RFC approval/merge process later today.

**Review by emalm — APPROVED** [review]

**Review by loewenstein — APPROVED** [review]

**Pull request merged**